slacker.py

```diff
@@ -64,24 +64,32 @@ def get_messages_in_time_range(self, oldest, cid, latest=None):
         cname = self.channels_by_id[cid]
         messages = []
         done = False
+        retry_attempts = 0
+        max_retry_attempts = 10
         while not done:
             murl = self.url + "channels.history?oldest={}&token={}&channel={}".format(oldest, self.token, cid)
             if latest:
                 murl += "&latest={}".format(latest)
             else:
                 murl += "&latest={}".format(int(time.time()))
             response = self.session.get(murl)
-            content = response.content
+
             try:
-                payload = response.json()
-            except ValueError as e:
```

Comment:
We were seeing these ValueErrors due to ignoring the response code... sometimes the rate limiting is a 429, sometimes a 503.